src/info/langs/language.rs

```diff
@@ -23,6 +23,8 @@ pub struct LanguagesInfo {
     number_of_languages_to_display: usize,
     #[serde(skip_serializing)]
     info_color: DynColors,
+    #[serde(skip_serializing)]
+    use_nerd_font_icons: bool,
```

Comment:
So this long option is `--use-nerd-font-icons`, right? I think that can be shortened to `--nerd-font`, and put "Use nerd font icons" in the help message. IDK how I feel about `-I` -- TBH I think this can be an option that has only a long form.

Reply:
renamed `--use-nerd-font-icons` to `--nerd-font`, updated the help message, and removed the short flag.